library/Ivoz/Ast/Domain/Service/PsEndpoint/UpdateByRetailAccount.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Ivoz\Ast\Domain\Service\PsEndpoint;
+
+use Ivoz\Ast\Domain\Model\PsEndpoint\PsEndpointRepository;
+use Ivoz\Core\Domain\Service\EntityPersisterInterface;
+use Ivoz\Ast\Domain\Model\PsEndpoint\PsEndpoint;
+use Ivoz\Ast\Domain\Model\PsEndpoint\PsEndpointInterface;
+use Ivoz\Provider\Domain\Model\RetailAccount\RetailAccountInterface;
+use Ivoz\Provider\Domain\Service\RetailAccount\RetailAccountLifecycleEventHandlerInterface;
+
+class UpdateByRetailAccount implements RetailAccountLifecycleEventHandlerInterface
+{
+    /**
+     * @var EntityPersisterInterface
+     */
+    protected $entityPersister;
+
+    /**
+     * @var PsEndpointRepository
+     */
+    protected $psEndpointRepository;
+
+    public function __construct(
+        EntityPersisterInterface $entityPersister,
+        PsEndpointRepository $psEndpointRepository
+    ) {
+        $this->entityPersister = $entityPersister;
+        $this->psEndpointRepository = $psEndpointRepository;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            self::EVENT_POST_PERSIST => 10
```

Comment:
Better use any LifecycleEventHandlerInterface priority constant instead of 10, self::PRIORITY_NORMAL for example